### ISSUE TITLE
Summarize other parts of HoTT in intro

### ISSUE THREAD

**Issue by mikeshulman** [HoTT/book#763]
Reading over the introduction again, I think there is something else missing: a brief summary of the other aspects of the field that the book is not about. The "Open problems" section makes some reference to these, but I think it may be more confusing than helpful to discuss open problems without having at least a very vague idea of the outlines of current knowledge.

**Comment by awodey** [comment]
the problem with this is that it’s easier to say what it *is* about than what it is *not* about, since the scope of the latter will keep changing as the field changes and develops.
if we add some remarks about what it is not about now, that will not accurately reflect what it was not about when it was written 2 years ago, nor what it is not about 2 years from now.
but OK, one can make some very general remarks about not being about proof theory, semantics, or formalization.

**Comment by mikeshulman** [comment]
Yes, it's a moving target, but I think since the book is many people's introduction to "homotopy type theory" we should give them _some_ idea of what that phrase means more generally than what's in the book.

**Comment by awodey** [comment]
OK with me.
will you draft something?

**Comment by mikeshulman** [comment]
On second thought, perhaps this would be too much of a change for the first edition.

**Comment by lombardisedr-dev** [comment]
Hi everyone ,i've been searching for a specifical channel,but i was not able to find something more appropriate...

PsiU-Protocol-HoTT: Semi-simplicial types & n-simplex stability via Univalence.
After 10 days of work, I’ve released this project on GitHub & Zenodo. It passed all my tests. I've been discovered since some years a logical talent,but I'm not an accademic one...so, I'd like to share my ideas, and inputs, or any kind of feedback - I'm lonely in my studies and I can't get over too long. Thanks!


https://github.com/lombardisedr-dev?tab=repositories

**Comment by mikeshulman** [comment]
This is not an appropriate place for that. A github issue is for discussing the particular question raised, not for asking for feedback on unrelated work.  You could try the homotopy type theory zulip server https://hott.zulipchat.com/.

**Comment by lombardisedr-dev** [comment]
Thanks,its my first day on GitHub,was not intentional.
